flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -50,6 +52,83 @@
 @SuppressWarnings("serial")
 public class StreamingJobGraphGeneratorTest extends TestLogger {
 
```

Comment:
I would suggest to add some tests (the same code as the existing ones) with invalid configurations and the expected exception.